debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorTask.java

```diff
@@ -171,14 +171,28 @@ protected Iterable<Field> getAllConfigurationFields() {
         return OracleConnectorConfig.ALL_FIELDS;
     }
 
-    private void validateRedoLogConfiguration() {
+    private void validateRedoLogConfiguration(OracleConnectorConfig config) {
         // Check whether the archive log is enabled.
         final boolean archivelogMode = jdbcConnection.isArchiveLogMode();
         if (!archivelogMode) {
-            throw new DebeziumException("The Oracle server is not configured to use a archive log LOG_MODE, which is "
-                    + "required for this connector to work properly. Change the Oracle configuration to use a "
-                    + "LOG_MODE=ARCHIVELOG and restart the connector.");
+            if (redoLogRequired(config)) {
+                throw new DebeziumException("The Oracle server is not configured to use a archive log LOG_MODE, which is "
+                        + "required for this connector to work properly. Change the Oracle configuration to use a "
+                        + "LOG_MODE=ARCHIVELOG and restart the connector.");
+            }
+            else {
+                LOGGER.warn("Failed the archive log check but continuing as redo log isn't strictly required");
+            }
         }
+        else {
+            LOGGER.info("Successfully validated the archive LOG_MODE");
+        }
```

Comment:
I think if archive log mode is enabled, there is no need to log anything.

Reply:
Fixed! Thank you.